pkgs/os-specific/linux/kernel/update-mainline.py

```diff
@@ -91,7 +91,7 @@ def main():
 
         print(message)
 
-        all_kernels[branch] = {"version": kernel.version, "hash": get_hash(kernel.link)}
+        all_kernels[branch] = {"version": kernel.version, "hash": "sha256:" + get_hash(kernel.link)}
```

Comment:
why not use sri hashes?

Reply:
Because I don't want to figure out how to convert nix-prefetch-url's base32 output to SRI, mostly...

Reply:
`nix --experimental-features nix-command hash to-sri --type sha256`

Reply:
I don't really want to shell out to another command just for this...

Reply:
fair enough. I just prefer that hash format, since it's a standard, but I won't die on this hill.

Reply:
I do too, but the overhead doesn't seem worth it here.